README.md

```diff
@@ -9,10 +9,10 @@
 
 > A note for Hacktoberfest Participants:
 >
-> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.
+> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT count for Hacktoberfest.
```

Comment:
Be counted works better here

Reply:
It is incorrect use of passive voice tho.

Reply:
I don’t think it’s wrong. The vote will be counted by some other subject so will be sounds better. Remember grammarly is a computer program that can’t always detect nuances so it’s not always correct either. As a native English speaker sometimes things just sound wrong to the ear, like it’s not natural and for me this is one of those times. In this case it’s not a massive issue but be wary in believing everything grammarly tells you ;-)

Reply:
```suggestion
> Pull requests which add or edit your information in a `data/${yourname}.json` file will NOT be counted for Hacktoberfest.
```